recipes/anaquin/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -eu -o pipefail
+
+# download and copy required dependencies
+# klib
+wget https://github.com/attractivechaos/klib/archive/spawn-final.tar.gz
```

Comment:
This is a pure library (i.e. no binaries):
https://github.com/attractivechaos/klib

Please note that I am downloading a github released tarball (i.e. a fixed version).

Moreover, it has not got updates since 2013 so I understood this is a very one off dependency. Do you think it is still good to create a bioconda package for it?